medusa/providers/torrent/json/t411.py

```diff
@@ -73,10 +89,90 @@ def __init__(self):
         # Cache
         self.cache = tv.Cache(self, min_time=10)  # Only poll T411 every 10 minutes max
 
+    def _get_season_search_strings(self, episode):
```

Comment:
I understand, but the common code should be moved into a separate method that each of those calls so you don't have to make the same changes in more than one place in the future.  It makes code more maintainable.